shared/AppInsightsCommon/src/StorageHelperFuncs.ts

```diff
@@ -88,142 +190,145 @@ export function utlEnableStorage() {
  * @param reset - Should the usage be reset and determined only based on whether LocalStorage is available
  */
 export function utlCanUseLocalStorage(reset?: boolean): boolean {
-    if (reset || _canUseLocalStorage === undefined) {
-        _canUseLocalStorage = !!_getVerifiedStorageObject(StorageType.LocalStorage);
+    if (reset || !_verifiedLocalStorage) {
+        _verifiedLocalStorage = createCachedValue(_getVerifiedStorageObject(StorageType.LocalStorage));
```

Comment:
Just have this call the existing helper to get the value, so that it's only done once